Data/DataModel/Submission.cs

```diff
@@ -73,6 +73,21 @@ public string SubmissionText
         /// </summary>
         public bool? FreeformAccepted { get; set; }
 
+        /// <summary>
```

Comment:
Most submissions are not freeform. How much of a perf issue do all of these extra columns add?

Reply:
Not a significant one. It's a few bytes per submission. It's possible we should have had this stuff in a side table, but I didn't realize we'd want more metadata when I started.